eras/conway/impl/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 1.9.0.0
 
+* Add `PredicateFailure` for `Voter` - `GovAction` mismatches, with `checkVotesAreValid`. #3718
```

Comment:
Maybe better to mention the name of the new data constructor `DisallowedVoters`  along with (or instead of) the explanation, because it's easier to search in future if we need to?

Reply:
It happens to be mentioned in the very next line, if I understand your suggestion correctly.

Reply:
Oh, of course! I got confused, sorry!